src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -185,6 +185,8 @@ public function dumpScalar(Cursor $cursor, $type, $value)
      */
     public function dumpString(Cursor $cursor, $str, $bin, $cut)
     {
+        $str = (string) $str;
```

Comment:
cannot be null

Reply:
You're right, didn't see the phpdoc on interface.
Fixed.